common/models/types/index.d.ts

```diff
@@ -50,6 +50,23 @@ declare interface LexiconTraversal {
    */
   children(): Generator<{char: USVString, traversal: () => LexiconTraversal}>;
 
+  /**
+   * Allows direct access to the traversal state that results when appending a
+   * `char` representing a single UTF-16 codepoint to the current traversal
```

Comment:
`single UTF-16 codepoint` is a little confusing on first read to me. Do you mean a `single codepoint encoded in UTF-16` (i.e. up to 2 UTF-16 code units?) 

I am not sure that `USVString` is really a thing in vanilla JavaScript/Typescript?

Reply:
I actually mean "one or more codepoints", which can be seen in #11870.  I'll rework to something that incorporates both this and that.

> I am not sure that USVString is really a thing in vanilla JavaScript/Typescript?

I'm not either, but we tagged a lot of other things that use strings within `common/models/types/index.d.ts` with USVString.  Just being consistent with "what is".  I never really understood the reason for the distinction being made there myself.

Reply:
* https://github.com/keymanapp/keyman/issues/12046